src/components/Founders/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+//importing images
+import Jen from '../../images/headshot-jen.jpg';
+import Therese from '../../images/headshot-therese.jpg';
+import Ruiting from '../../images/headshot-ruiting.jpg';
+//import CSS
+import './styles.css';
+
+const FoundersIndexPage = () => (
+	<div>
+		<div class="Founders-div">
+			<h1 id="Founders-h1">The Founders</h1>
+		</div>
+		<div class="Founders-div">
+			<div>
+				<img class="Founders-img" src={Jen} alt="Jen" />
```

Comment:
Typically classes would be lower case. like founders-img instead of "Founders-img"